index_V1

```diff
@@ -31,6 +43,14 @@
 				
 				<div class="table">
 					
+					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 1; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					
+					<div onclick="location.href='query_1.php';"class="circle_2" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					
+					<div onclick="location.href='query_1.php';"class="circle_3" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+					
+					<div onclick="location.href='query_1.php';"class="circle_4" style="cursor: pointer; grid-column: 4; background-color: #e6e6e6; border-radius: 30vw; margin: 0vw 1.5vw; transition: 0.6s;"><p style="margin: 15vw 3.5vw; transition: 0.6s;">Put Link Here</p></div>
+				
```

Comment:
Here I created the divs for the four circles which will direct users to a certain query.